media-proxy/src/proxy_context.cc

```diff
@@ -758,7 +776,7 @@ int ProxyContext::RxStart(const mcm_conn_param* request)
         ParseStInitParam(request, &st_param);
 
         /* set default parameters */
-        // st_param.flags = MTL_FLAG_BIND_NUMA;
+        st_param.flags = MTL_FLAG_BIND_NUMA;
```

Comment:
This seems not related to the PR scope. Could you elaborate more? Why you enable this flag?

Reply:
In multi-numa environments this flags try to assign cores correctly in regards to used interface hardware numa number, basically allowing speed boost. This was mandatory for the presentation.

Reply:
Maybe you could amend the commit message to cover that change? But it's up to you.

Reply:
I can add this to the description of PR as this flag was added in first commit that was done.